cmd/cspc-operator/app/handler.go

```diff
@@ -119,6 +122,12 @@ func (c *Controller) syncCSPC(cspcGot *apis.CStorPoolCluster) error {
 		return nil
 	}
 
+	_, err := c.populateVersion(cspcGot)
```

Comment:
you better take the object returned here

Reply:
I am thinking to take this func below as this func can return nil cspc resource in case of any error, we should be breaking the provisioning flow IMO ?

Reply:
taken return object